graphql.go

```diff
@@ -65,7 +64,12 @@ func (c *Client) do(ctx context.Context, op operationType, v interface{}, variab
 	if err != nil {
 		return err
 	}
-	resp, err := ctxhttp.Post(ctx, c.httpClient, c.url, "application/json", &buf)
+	req, err := http.NewRequestWithContext(ctx, "POST", c.url, &buf)
```

Comment:
I prefer using the `http.MethodPost` constant as your original version had.

(I'll take care of it as part of merging.)